src/plugins/visualizations/public/plugin.ts

```diff
@@ -103,6 +119,8 @@ export interface VisualizationsStartDeps {
   getAttributeService: EmbeddableStart['getAttributeService'];
   savedObjects: SavedObjectsStart;
   savedObjectsClient: SavedObjectsClientContract;
+  spaces: SpacesPluginStart;
```

Comment:
I see the following dependencies should be optional
```
spaces?: SpacesPluginStart;
savedObjectsTaggingOss?: SavedObjectTaggingOssPluginStart;
```

Reply:
done